### ISSUE TITLE
Sample does not connect when pairing is required

### ISSUE THREAD

**Issue by darrinps** [RobotPajamas/Blueteeth#48]
I have a device that requires you to pair to it. You cannot pair ahead of time as it stops broadcasting once it is paired. It would be great if the sample were made to work like BLE Scanner (available on Google Play) that pairs then connects.

**Comment by sureshjoshi** [comment]
I'm planning to beef these up and cover some of the outstanding bugs over the Christmas holidays, when I free up. 

Proper pairing is high on my list.

**Comment by darrinps** [comment]
If there is anything I can do to help out (test beta builds, etc.) please let me know.

**Comment by sureshjoshi** [comment]
Yeah, I would appreciate that - I've recently culled my Android test phones from about 10 to 3, so the more testing, the merrier.

In a week or so, I'll start making commits and link this issue, so you'll be able to see what's going on.
